docs/getting_started/installation.rst

```diff
@@ -218,11 +197,13 @@ You've finished the installation process, congratulations. If you fire up the
 server, you can now visit the admin page on http://localhost:8000/admin
 
 .. note::
-
     This tutorial assumes you are using the build-in server using the
     ``php bin/console server:start`` (or ``server:run``) command.
 
-.. image:: ../images/getting_started_empty_dashboard.png
+.. figure:: ../images/getting_started_empty_dashboard.png
```

Comment:
This makes the image not show like this: https://sonata-project.org/bundles/admin/3-x/doc/getting_started/installation.html and be like this: https://symfony.com/doc/master/bundles/SonataAdminBundle/reference/security.html